### PR TITLE
[PDI-19906] - Google BigQuery Simba driver version 1.3.3.1004 in Pentaho

### PR DESCRIPTION
**Pull request by bcostahitachivantara**

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 45s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 2m 7s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 43s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 48s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 3m 17s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

File: pom.xml [hunk 1]
```diff
@@ -374,9 +374,6 @@
     <rome.version>1.18.0</rome.version>
     <rhino.version>1.7.13</rhino.version>
     <snakeyaml.version>1.33</snakeyaml.version>
-    <sqlite-jdbc.version>3.41.2.2</sqlite-jdbc.version>
```
Comment (lucboudreau) [inline pom.xml:377]: Are you sure you want to delete these 3? This was just put in 4 days ago. Maybe it's a merge error?

https://github.com/pentaho/maven-parent-poms/blame/master/pom.xml#L377-L379

File: pom.xml [hunk 2]
```diff
@@ -414,7 +411,7 @@
       <dependency>
         <groupId>org.eclipse.jetty</groupId>
         <artifactId>jetty-server</artifactId>
-        <version>${jetty-server.version}</version>
+        <version>${jetty.version}</version>
```
Comment (lucboudreau) [inline pom.xml:414]: Same comment here. This was done 3 days ago. Are you sure this needs to be deleted?

https://github.com/pentaho/maven-parent-poms/blame/master/pom.xml#L417

File: pom.xml (hunk 1 above)
Comment (bcostahitachivantara) [reply pom.xml:377]: You are right, let me fix that

File: pom.xml (hunk 2 above)
Comment (bcostahitachivantara) [reply pom.xml:414]: You are right, let me fix that

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 2m 11s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 53s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Review by smmribeiro — APPROVED** [review]
Looks good!

**Pull request merged**